docker-compose.yml

```diff
@@ -69,4 +85,19 @@ services:
       INFLUXURL: http://influxdb:8086
       NUOCMD_API_SERVER: https://nuoadmin1:8888
       NUOCD_HOSTNAME: nuoadmin1
-    pid: 'service:nuoadmin1'
\ No newline at end of file
+      INFLUXDB_TOKEN: $DOCKER_INFLUXDB_INIT_ADMIN_TOKEN
+      INFLUXDB_BUCKET: $DOCKER_INFLUXDB_INIT_BUCKET
+      INFLUXDB_ORG: $DOCKER_INFLUXDB_INIT_ORG
+    pid: 'service:nuoadmin1'
+
+  ycsb-demo:
+    image: nuodb/ycsb:latest
+    depends_on:
+      - te
+      - nuocd-te
+    environment:
+      PEER_ADDRESS: nuoadmin1
+      DB_NAME: hockey
+      DB_USER: dba
+      DB_PASSWORD: goalie
+    command: ["/driver/startup.sh"]
```

Comment:
Have you been able to identify if the lack of_synctrace_ metrics in the idle database is due to the change in NuoDB version?

Reply:
The NuoDB version did not change in this PR.